src/main/java/edu/harvard/iq/dataverse/workflow/internalspi/AuthorizedExternalStep.java

```diff
@@ -1,6 +1,7 @@
 package edu.harvard.iq.dataverse.workflow.internalspi;
 
 import edu.harvard.iq.dataverse.util.BundleUtil;
```

Comment:
Another comment near the top.

@aliassheikh it's awesome that you added a unit test but can you please also update the "how to test" part of the description for how this should be tested when in QA? Thanks!